package.json

```diff
@@ -27,7 +27,7 @@
     "cache-manager-fs": "^1.0.5",
     "cache-manager-fs-binary": "^1.0.2",
     "cache-manager-hazelcast": "^0.2.1",
-    "cache-manager-mongodb": "^0.1.6",
+    "cache-manager-mongodb": "v4l3r10/node-cache-manager-mongodb",
```

Comment:
If you wish to get the last commits from cache manager mongodb please ask them to update, tag and publish to npm as 0.1.7 or anything else because i wont put the master of this projet into deps. I only add tags that has been published which that freeze the component and depedencies.
In fact to ensure package keep working we have to be sure that deps are stable too.